__tests__/middlewares.test.js

```diff
@@ -0,0 +1,16 @@
+test("Should return a JWT with proper value if nothing wrong happened", async () => {
+  const result = await new Promise(resolve => {
+    issueJWT(request, response, err => {
+      if (!err) {
+        resolve(response.locals.token);
+      }
+    });
+  });
+
+  const tokenPayload = jwt.decode(result, { complete: true }).payload;
+
+  expect(tokenPayload).toHaveProperty("iat");
+  expect(tokenPayload).toHaveProperty("exp");
+  expect(tokenPayload).toHaveProperty("id");
+  expect(tokenPayload).toHaveProperty("iss");
+});
```

Comment:
# Nitpick

Add a new line after the last line of every file. Check [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for context.

This should be captured in an `editorconfig` file though

Reply:
I edited the .editorconfig file. It should be ok now.